core/controllers/classroom_test.py

```diff
@@ -82,13 +82,15 @@ def test_get(self):
         csrf_token = self.get_new_csrf_token()
         new_config_value = [{
             'name': 'math',
-            'topic_ids': [topic_id_1, topic_id_2]
+            'topic_ids': [topic_id_1, topic_id_2],
+            'course_details': 'Course details for classroom.',
+            'topics_covered': 'Topics covered for classroom'
```

Comment:
topics_covered sounds like a list. Maybe give it a more "string"-sounding name, like "description"? (Not sure what it's supposed to represent.)

Reply:
It's just supposed to give an introduction before the topic icons are showed. I'll rename it to `topics_covered_introduction`

Reply:
Oh, I see. I think `topic_list_intro` would be a fine name (it's an introduction to the topic list).

Reply:
Done